src/core_modules/capture-core/components/DataEntries/Enrollment/EnrollmentDataEntry.component.js

```diff
@@ -312,8 +312,9 @@ class PreEnrollmentDataEntryPure extends React.PureComponent<Object> {
 
 export class EnrollmentDataEntryComponent extends React.Component<PreEnrollmentDataEntryProps> {
     getValidationContext = () => {
-        const { orgUnit, onGetUnsavedAttributeValues } = this.props;
+        const { orgUnit, onGetUnsavedAttributeValues, programId } = this.props;
         return {
+            scope: programId,
```

Comment:
same as other comment:

(I think I would prefer to use the prop name programId here. This is always a program id and I don't think there is a reason the generalise here. Let me know what you think.)